src/main/java/com/sfuentes/Farm/Farm.java

```diff
@@ -0,0 +1,30 @@
+package com.sfuentes.Farm;
+
+import java.util.List;
+
+public class Farm {
+
+  public Farm() {
+  }
+
+  public double getQualityByHen(Hen hen) {
+    return hen.getHenWeight() * hen.getHenHeight() / hen.getNumberOfEggs();
+  }
+
+  public double getEggPriceByKilo(Hen hen) {
+    double price;
+
+    if (getQualityByHen(hen) >= 15)
+      price = 12d;
+    else if (getQualityByHen(hen) > 8 && getQualityByHen(hen) < 15)
```

Comment:
There is no need to evaluate if quality is les than 15 since this is an else if and in the first one  the condition  already fell into the else which means quality IS less than 15

Reply:
fix